clients/src/main/java/org/apache/kafka/clients/consumer/internals/CommitRequestManager.java

```diff
@@ -454,6 +491,17 @@ public OffsetFetchRequestState(final Set<TopicPartition> partitions,
             this.future = new CompletableFuture<>();
         }
 
+        public OffsetFetchRequestState(final Set<TopicPartition> partitions,
+                                       final GroupState.Generation generation,
+                                       final long retryBackoffMs,
+                                       final long retryBackoffMaxMs,
+                                       final double jitter) {
+            super(logContext, CommitRequestManager.class.getSimpleName(), retryBackoffMs, 2, retryBackoffMaxMs, jitter);
```

Comment:
@lucasbru - Just FYI: I added this so that exponential backoff during testing doesn't have jitter.